mochify/test/files.integration.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const { assert, sinon } = require('@sinonjs/referee-sinon');
+const { mochify } = require('..');
+
+describe('files', () => {
+  it('passes files to bundle command', async () => {
+    sinon.replace(process.stdout, 'write', sinon.fake());
+
+    await mochify({
+      driver: 'jsdom',
+      reporter: 'json',
+      files: [`${__dirname}/fixture/passes.js`],
+      bundle: 'cat'
```

Comment:
I assume this will make the tests fail on Windows, but I guess that's ok?

Reply:
Good point. Any idea for an alternative?

Reply:
Hmm, `cat` seems to work in Powershell, but not in `cmd.exe`: https://stackoverflow.com/a/52388383/797194

Not entirely sure if we want to roll our own or even install a third-party package tbh.

Reply:
Let's keep it like this until it's actually someone's problem.